web/packages/build/package.json

```diff
@@ -15,6 +15,7 @@
     "@babel/preset-react": "^7.27.1",
     "@babel/preset-typescript": "^7.27.1",
     "@eslint/js": "^9.30.0",
+    "@eslint/plugin-kit": "^0.3.3",
```

Comment:
You need to remove this dependency and run `pnpm install` again. The upgraded version will be kept in the lockfile.